malariagen_data/anoph/sample_metadata.py

```diff
@@ -1183,6 +1183,155 @@ def cohorts(
 
         return df_cohorts
 
+    @check_types
+    @doc(
+        summary="""
+            Plot markers on a map showing sample locations
+            as a Mapbox scatter plot.
+        """,
+        parameters=dict(
+            kwargs="Passed through to px.scatter_mapbox().",
+        ),
+    )
+    def sample_location_mapbox(
```

Comment:
```suggestion
    def plot_sample_location_mapbox(
```

Naming suggestion for consistency with other functions, "plot_" sets expectation the function will generate some kind of figure.